apps/frontend/src/pages/profile/overview.tsx

```diff
@@ -83,17 +89,37 @@ export default function ProfileOverview() {
             {user?.userData.telegram?.username && (
               <LinkCardBox
                 label="Telegram"
-                value={user?.userData.telegram.username}
+                value={`@${user?.userData.telegram.username}`}
                 href={`https://t.me/${user.userData.telegram.username}`}
               />
             )}
             {user?.userData.twitter?.username && (
               <LinkCardBox
                 label="Twitter"
-                value={user?.userData.twitter.username}
+                value={`@${user?.userData.twitter.username}`}
                 href={`https://twitter.com/${user.userData.twitter.username}`}
               />
             )}
+            {user?.userData.signal?.username && (
+              <CardBox
+                label="Signal"
+                value={`@${user?.userData.signal.username}`}
```

Comment:
Link to signal?

Reply:
Couldn't find the username link, will look again

Reply:
sgnl://signal.me/#u/stevenelleman.87 may work